apps/fabric-website/src/root.tsx

```diff
@@ -145,4 +145,4 @@ function addCSSToHeader(fileName: string) {
   headEl.appendChild(linkEl);
 }
 
-addCSSToHeader('https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/9.0.0/css/fabric.min.css');
+addCSSToHeader('https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/' + corePackageData.version + '/css/fabric.min.css');
```

Comment:
This seems like it could be fragile. Can we guarantee that if a new fabric-core dependency is released, it will be available on this path? With this exact version?

Reply:
Added some safe checks for the const, let me know if it's still too fragile